transifex/native/django/management/common.py

```diff
@@ -39,6 +39,11 @@ class SourceStringCollection(object):
     def __init__(self):
         self.strings = {}
 
+    def serialize_strings(self):
+        """Serializes all string based on the APIv3 specs
+
+        """
+
```

Comment:
?

Reply:
@kbairak that was to serialize the occurrences into the API3 format. I think I leave them alone and serialize them in the `cds` part.